src/katgpucbf/xbgpu/engine.py

```diff
@@ -892,11 +859,11 @@ async def _receiver_loop(self) -> None:
         Receive heaps off of the network in a continuous loop.
 
         This function does the following:
-        1. Wait for a chunk to be assembled on the receiver.
-        2. Get a free rx item off of the _rx_free_item_queue.
-        3. Initiate the transfer of the chunk from system memory to the buffer
-           in GPU RAM that belongs to the rx_item.
-        4. Place the rx_item on _rx_item_queue so that it can be processed downstream.
+        - Wait for a chunk to be assembled on the receiver.
+        - Get a free rx item off of the _rx_free_item_queue.
+        - Initiate the transfer of the chunk from system memory to the buffer
```

Comment:
I'm not so sure I like conversion of all the numbered lists to bullet lists. Numbered lists imply that things are happening in a specific order, while this bullet list could be taken as an unordered collection.

This documentation doesn't seem to get rendered (presumably because it is a private member), but I think you need a blank line before the list starts to make it correct.

Reply:
I'm absolutely with you; I was (still am) a big proponent of the numbered list. I changed it because I recall (way back) we had a discussion around numbered-list comments and general discomfort around maintaining/keeping them in parity with what the code is actually doing.

Will roll back and update to add a blank line.

Reply:
Yes, there is a balance. But I think that discussion was more about the numbered comments sprinkled through the code, where adding/removing a step would require renumbering things at distant points, and there were also multiple levels of numbering. With this docstring the list is more self-contained.